blockchain/chainquery.go

```diff
@@ -137,9 +137,7 @@ func (b *BlockChain) ChainTips() []ChainTipInfo {
 // BestHeader returns the header with the most cumulative work that is NOT
 // known to be invalid.
```

Comment:
Unrelated to what you changed, but this function comment could also state that the function is safe for concurrent access.

Reply:
Agreed.  I'll open a PR up for it later tonight.